extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -78,6 +130,25 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		});
 	}
 
+	async openNotebookAsUntitled(resource: string): Promise<void> {
```

Comment:
Why is this async? Would also suggest removing the .then and awaiting on openTextDocument instead

Reply:
removed async